packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -523,6 +523,9 @@ export const TombstoneResponseHeaderKey = "isTombstoned";
  * Inactive error responses will have this header set to true
  * @legacy
  * @alpha
+ *
+ * @deprecated - This header is deprecated and will be removed in the future. The functionality corresponding
```

Comment:
Dash is unnecessary for `@deprecated`, it would show up as part of the text.
```suggestion
 * @deprecated This header is deprecated and will be removed in the future. The functionality corresponding
```